--- configure.scan.old	2011-02-28 13:46:52.262000002 +0100
+++ configure.scan	2011-02-28 13:47:53.250000002 +0100
@@ -2,16 +2,20 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.68])
-AC_INIT([FULL-PACKAGE-NAME], [VERSION], [BUG-REPORT-ADDRESS])
+AC_INIT(2lgc_code, 0.0.0.2, devel@2lgc.org, 2lgc_code, http://2lgc.org)
+AM_INIT_AUTOMAKE
 AC_CONFIG_SRCDIR([src/test/1990__1.c])
 AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_MACRO_DIR([m4])
 
 # Checks for programs.
 AC_PROG_CC
+AC_PROG_LIBTOOL
+AM_GNU_GETTEXT([external])
+ALL_LINGUAS="fr"
 
 # Checks for libraries.
-# FIXME: Replace `main' with a function in `-llist':
-AC_CHECK_LIB([list], [main])
+PKG_CHECK_MODULES([LIBLIST], [liblist >= 2.3.1], [], [AC_MSG_ERROR([liblist ( http://ohnopub.net/liblist ) est necessaire])])
 
 # Checks for header files.
 AC_CHECK_HEADERS([libintl.h locale.h stdlib.h string.h])
