--- configure.scan.old	2011-07-04 16:33:28.649007767 +0200
+++ configure.scan	2011-07-04 16:33:11.590007763 +0200
@@ -2,20 +2,39 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.68])
-AC_INIT([FULL-PACKAGE-NAME], [VERSION], [BUG-REPORT-ADDRESS])
+AC_INIT(2lgc_code, 0.0.0.2, devel@2lgc.org, 2lgc_code, http://2lgc.org)
+AM_INIT_AUTOMAKE
 AC_CONFIG_SRCDIR([src/test/1990__1_duree.c])
 AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_MACRO_DIR([m4])
 
 # Checks for programs.
 AC_PROG_CC
+AM_PROG_CC_C_O
+AC_PROG_LIBTOOL
+LIBXML_REQUIRED_VERSION=2.7.0
+AC_SUBST(XML_CPPFLAGS)
+AC_SUBST(XML_LIBS)
+AM_PATH_XML2($LIBXML_REQUIRED_VERSION, :, AC_MSG_ERROR(Test for libXML failed.))
+AC_ARG_WITH(gtk,  [  --without-gtk           build without gtk support], [withgtk=true], [withgtk=false])
+AC_MSG_RESULT([XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+with-gtk $withgtk
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX])
+AM_CONDITIONAL(ENABLE_GTK, test x$withgtk = xtrue)
+if test "x$withgtk" = "xtrue"; then
+GTK_REQUIRED_VERSION=3.0.3
+GLIB_REQUIRED_VERSION=2.28.3
+AM_PATH_GTK_3_0($GTK_REQUIRED_VERSION, :, AC_MSG_ERROR(Test for GTK+ failed.))
+AM_PATH_GLIB_2_0($GLIB_REQUIRED_VERSION, :, AC_MSG_ERROR(Test for GLIB failed.))
+AC_DEFINE(ENABLE_GTK, 1, [Défini si GTK doit être supporté])
+fi
+AM_GNU_GETTEXT([external])
+ALL_LINGUAS="fr"
 
 # Checks for libraries.
-# FIXME: Replace `main' with a function in `-lcholmod':
-AC_CHECK_LIB([cholmod], [main])
-# FIXME: Replace `main' with a function in `-llist':
-AC_CHECK_LIB([list], [main])
-# FIXME: Replace `main' with a function in `-lspqr':
-AC_CHECK_LIB([spqr], [main])
+PKG_CHECK_MODULES([LIBLIST], [liblist >= 2.3.1], [], [AC_MSG_ERROR([liblist ( http://ohnopub.net/liblist ) est necessaire])])
+AC_CHECK_LIB([cholmod], [cholmod_l_start], [], [AC_MSG_ERROR([cholmod ( http://www.cise.ufl.edu/research/sparse/cholmod ) est necessaire])])
+AC_CHECK_LIB([spqr], [SuiteSparseQR_C_factorize], [], [AC_MSG_ERROR([spqr ( www.cise.ufl.edu/research/sparse/SPQR ) est necessaire])])
 
 # Checks for header files.
 AC_CHECK_HEADERS([libintl.h locale.h stdlib.h string.h unistd.h values.h])
