--- po/Makevars.template.old	2011-02-21 16:42:11.106000001 +0100
+++ po/Makevars.template	2011-02-21 16:43:21.179000001 +0100
@@ -11,1 +11,1 @@ subdir = po
-XGETTEXT_OPTIONS = --keyword=_ --keyword=N_
+XGETTEXT_OPTIONS = --keyword=i18n --keyword=i18nM --keyword=i18nP:1,2 --from-code=utf-8 --no-location --no-wrap
@@ -21,1 +21,1 @@ XGETTEXT_OPTIONS = --keyword=_ --keyword
-COPYRIGHT_HOLDER = Free Software Foundation, Inc.
+COPYRIGHT_HOLDER = 2LGC
@@ -44,1 +44,1 @@ COPYRIGHT_HOLDER = Free Software Foundat
-MSGID_BUGS_ADDRESS =
+MSGID_BUGS_ADDRESS = devel@2lgc.org
