--- configure.scan.old	2011-02-08 16:44:11.186000001 +0100
+++ configure.scan	2011-02-08 16:45:47.649000001 +0100
@@ -2,12 +2,18 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.68])
-AC_INIT([FULL-PACKAGE-NAME], [VERSION], [BUG-REPORT-ADDRESS])
+AC_INIT(2lgc_code, 0.0.0.1, devel@2lgc.org, 2lgc_code, http://2lgc.org)
+AM_INIT_AUTOMAKE
 AC_CONFIG_SRCDIR([src/gui/main.c])
 AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_MACRO_DIR([m4])
+
 
 # Checks for programs.
 AC_PROG_CC
+AC_PROG_LIBTOOL
+AM_GNU_GETTEXT([external])
+ALL_LINGUAS="fr"
 
 # Checks for libraries.
 
