--- configure.scan.old	2011-09-10 19:40:15.109918470 +0200
+++ configure.scan	2011-09-10 19:40:05.964918617 +0200
@@ -5,1 +5,2 @@
-AC_INIT([FULL-PACKAGE-NAME], [VERSION], [BUG-REPORT-ADDRESS])
+AC_INIT(2lgc_code, 0.0.0.2, devel@2lgc.org, 2lgc_code, http://2lgc.org)
+AM_INIT_AUTOMAKE
@@ -8,24 +10,46 @@
+AM_GNU_GETTEXT([external])
+AC_CONFIG_MACRO_DIR([m4])
 
 # Checks for programs.
 AC_PROG_AWK
 AC_PROG_CXX
 AC_PROG_CC
 AC_PROG_MAKE_SET
+AM_PROG_CC_C_O
+AC_PROG_LIBTOOL
+LIBXML_REQUIRED_VERSION=2.7.0
+AC_SUBST(XML_CPPFLAGS)
+AC_SUBST(XML_LIBS)
+AM_PATH_XML2($LIBXML_REQUIRED_VERSION, :, AC_MSG_ERROR(Test for libXML failed.))
+AC_ARG_WITH(gtk,  [  --without-gtk           build without gtk support], [withgtk=true], [withgtk=false])
+AC_MSG_RESULT([XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+with-gtk $withgtk
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX])
+AM_CONDITIONAL(ENABLE_GTK, test x$withgtk = xtrue)
+if test "x$withgtk" = "xtrue"; then
+GTK_REQUIRED_VERSION=3.0.3
+GLIB_REQUIRED_VERSION=2.28.3
+AM_PATH_GTK_3_0($GTK_REQUIRED_VERSION, :, AC_MSG_ERROR(Test for GTK+ failed.))
+AM_PATH_GLIB_2_0($GLIB_REQUIRED_VERSION, :, AC_MSG_ERROR(Test for GLIB failed.))
+AC_DEFINE(ENABLE_GTK, 1, [Défini si GTK doit être supporté])
+fi
+AC_ARG_WITH(debug,  [  --without-debug       build without debug support], [withdebug=true], [withdebug=false])
+AC_MSG_RESULT([XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+with-debug $withdebug
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX])
+if test "x$withdebug" = "xfalse"; then
+AC_DEFINE(NDEBUG, 1, [Désactive le debogage])
+fi
+AM_GNU_GETTEXT([external])
+ALL_LINGUAS="fr"
 
 # Checks for libraries.
-# FIXME: Replace `main' with a function in `-lamd':
-AC_CHECK_LIB([amd], [main])
-# FIXME: Replace `main' with a function in `-lblas':
-AC_CHECK_LIB([blas], [main])
-# FIXME: Replace `main' with a function in `-lcholmod':
-AC_CHECK_LIB([cholmod], [main])
-# FIXME: Replace `main' with a function in `-lcolamd':
-AC_CHECK_LIB([colamd], [main])
-# FIXME: Replace `main' with a function in `-lgfortran':
-AC_CHECK_LIB([gfortran], [main])
-# FIXME: Replace `main' with a function in `-llist':
-AC_CHECK_LIB([list], [main])
-# FIXME: Replace `main' with a function in `-lumfpack':
-AC_CHECK_LIB([umfpack], [main])
+PKG_CHECK_MODULES([LIBLIST], [liblist >= 2.3.1], [], [AC_MSG_ERROR([liblist ( http://ohnopub.net/liblist ) est necessaire])])
+AC_CHECK_LIB([amd], [amd_malloc], [], [AC_MSG_ERROR([amd ( http://www.cise.ufl.edu/research/sparse/amd ) est necessaire])])
+AC_CHECK_LIB([gfortran], [_gfortran_st_write], [], [AC_MSG_ERROR([ gfortran est necessaire ])])
+AC_CHECK_LIB([blas], [dtrsv_], [], [AC_MSG_ERROR([blas ( http://netlib.org/lapack ) est necessaire])])
+AC_CHECK_LIB([cholmod], [cholmod_l_start], [], [AC_MSG_ERROR([cholmod ( http://www.cise.ufl.edu/research/sparse/cholmod ) est necessaire])])
+AC_CHECK_LIB([colamd], [colamd_l_recommended], [], [AC_MSG_ERROR([colamd ( www.cise.ufl.edu/research/sparse/colamd ) est necessaire])])
+AC_CHECK_LIB([umfpack], [umfpack_dl_solve], [], [AC_MSG_ERROR([umfpack ( www.cise.ufl.edu/research/sparse/umfpack ) est necessaire])])
 
 # Checks for header files.
