--- Makefile.rule.old	2012-08-20 08:52:35 +0000
+++ Makefile.rule	2012-09-01 20:01:53 +0000
@@ -19,23 +19,23 @@ VERSION = 0.2.3
 
 # C compiler including binary type(32bit / 64bit). Default is gcc.
 # Don't use Intel Compiler or PGI, it won't generate right codes as I expect.
-# CC = gcc
+CC = gcc
 
 # Fortran compiler. Default is g77.
-# FC = gfortran
+FC = gfortran
 
 # Even you can specify cross compiler
 # CC = x86_64-w64-mingw32-gcc
 # FC = x86_64-w64-mingw32-gfortran
 
 # If you need 32bit binary, define BINARY=32, otherwise define BINARY=64
-# BINARY=64
+BINARY=32
 
 # About threaded BLAS. It will be automatically detected if you don't
 # specify it.
 # For force setting for single threaded, specify USE_THREAD = 0
 # For force setting for multi  threaded, specify USE_THREAD = 1
-# USE_THREAD = 0
+USE_THREAD = 1
 
 # If you're going to use this library with OpenMP, please comment it in.
 # USE_OPENMP = 1
@@ -53,7 +53,7 @@ VERSION = 0.2.3
 # NO_LAPACK = 1
 
 # If you don't need LAPACKE (C Interface to LAPACK), please comment it in.
-# NO_LAPACKE = 1
+NO_LAPACKE = 1
 
 # If you want to use legacy threaded Level 3 implementation.
 # USE_SIMPLE_THREADED_LEVEL3 = 1
@@ -106,17 +106,17 @@ VERSION = 0.2.3
 # UTEST_CHECK = 1
 
 # The installation directory.
-# PREFIX = /opt/OpenBLAS
+PREFIX = c:/MinGW
 
 # Common Optimization Flag; 
 # The default -O2 is enough.
-# COMMON_OPT = -O2
+COMMON_OPT = -march=i686 -O0 -g3 -ggdb3 -pipe
 
 # Profiling flags
 COMMON_PROF = -pg
 
 # Build Debug version
-# DEBUG = 1
+DEBUG = 1
 
 #
 #  End of user configuration 
